src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -657,6 +684,30 @@ class Browser extends DashboardView {
       query.ascending(field)
     }
 
+    if (field !== 'objectId') {
```

Comment:
This seems to be a merge issue; this code does not exist in the master branch anymore, see https://github.com/parse-community/parse-dashboard/pull/1800.

I am worried that there is some code in this PR that is obsolete but is still here due to master merging. Can you please take a minute and look through tall the change in this PR, compare it to the current master, and see if all the changes are related to your PR? Sorry for the extra work, but we better take an extra look before we introduce a bug.